### DISCUSSION TITLE
Move to GitHub organisation

### DISCUSSION THREAD

**Discussion by axelboc** [axelboc/anki-ultimate-geography#373] [Ideas]
Now that Brain Brew has been adopted as our new deck builder, I think the next step is to move this repo to a GitHub organisation. Any [naming suggestions](https://github.com/account/organizations/new?coupon=&plan=team_free)?

- ~ug~ :no_entry: 
- ultimate-geography ✅
- ultimate-geo ✅
- anki-ug ✅
- anki-geo ✅
- anki-geography ✅
- brain-brew ✅ 😮

My personal favourite because it's short: **anki-ug**.

**Comment by aplaice** [comment]
`anki-ug` is indeed quite nice!

OTOH, if we want to leave ourselves room to be ambitious and *possibly*, *eventually* lay claim to all of Geography for Anki, with the idea that there'd be other geography decks managed here, with similar tools (not necessarily personally by us...), not just AUG and its "extensions", then `anki-geography` (or `anki-geo`) would be also great!

> brain-brew white_check_mark open_mouth

That's a provocative suggestion! (I understand that the idea is that @ohare93's `brain-brew` would also be moved there and `anki-ultimate-geography` would become an official "flagship" project?) I'm 100% for close collaboration between `brain-brew` and `aug`, but I don't think that combining them under one umbrella would be optimal for either. After all, `brain-brew` is supposed to be about more than geography decks, while `aug` relies on several other essential tools (`CrowdAnki`, `Anki` itself). I see relatively little upside and potential for some downside in case of diverging needs, if that were to happen in the future.

**Comment by jeprecated** [comment]
> brain-brew white_check_mark open_mouth

You'll be hearing from my lawyer! 😆 Nah but seriously, what @aplaice said 😉 

> anki-ug

I read this as "Anki Uhg" as in the type of boots 😅 I know it is supposed to be "Anki U-Gee" so I am not opposed to it, as long as it is capitalised! 👍 

How about: UGG (Ultimate Geography Group) 😄 It maintains the original known name while adding to it too. Thesaurus the word Group for anything else 🙂 (UGF?! 😉 engage the Federation!)

**Comment by axelboc** [comment]
`UGG` is taken and so is `UGF`, `AUG`, even `UGG...G` up to ... 7 Gs! 😂 

On one hand, `anki-geo` does seem promising, then, if we decide to broaden our horizons. On the other hand, it would make UG's core and extension repositories more challenging, as they will have to include the name `ultimate-geography` or `ug` or similar.

If we decide that the organisation should be dedicated to Ultimate Geography and its extensions, then we can just have a repository called `standard` or `core` and similarly simple names for extensions.

**Reply by axelboc** [reply]
By the way, we could go all the way and just use `anki-decks`, in which case including the brain brew repo would make sense. 🤷‍♂️ Maybe it's too crazy, I don't know 😅.

**Reply by axelboc** [reply]
Oh damn, it's taken! 😂 https://github.com/anki-decks

**Reply by jeprecated** [reply]
> Oh damn, it's taken! joy https://github.com/anki-decks

I was expecting more from such a good name! :sweat_smile:

**Reply by axelboc** [reply]
Maybe we can ask to be added to the org? 🤷 Ah but not if we don't want `anki` in the name, of course.

**Comment by jeprecated** [comment]
I don't know if having "anki" in the name is the best way to go :slightly_frowning_face: after all, the information here is (and should be) tool agnostic. Anki may be the only thing that this deck is being used with just now, but perhaps not in the future :shrug: 

Thoughts?

**Reply by axelboc** [reply]
Very true!

**Reply by aplaice** [reply]
Since it seems that we don't want to be constrained either to geography or Anki, are you suggesting that we claim `srs-decks`? :D 

I'm not sure, though, that decoupling from Anki should be a primary aim.

At the moment, the largest Anki-specific part of the "data" in the repo is the templates (if I remember correctly both Supermemo's and Mnemosyne's\* templates are far more XML-heavy, and are not, in any case, directly compatible with Anki's).  However, if it turns out that we could improve the learning experience on Anki, at the cost of further increasing "distance" from other SRSs, I'd say we should definitely do it.

\* I'm listing the main SRSs that I'm aware of

For instance, if, hypothetically, the functionality of the "link notes" addons (e.g. [this](https://ankiweb.net/shared/info/1423933177) or [this](https://ankiweb.net/shared/info/1126950429)) was incorporated into Anki/Ankimobile/Ankidroid (and made more compatible with CrowdAnki), I'd argue that adding cross-note links for the flag similarity cards would be reasonable to do, even though it'd be useless for all other SRS platforms.

That's not to say that I wouldn't welcome compatibility with other SRS programs, if somebody™ implemented and maintained it. However, even then having Anki in the name of the organisation wouldn't really be out of place. If FreeBSD can use ZFS-on-Linux, then Mnemosyne/Supermemo can import an Anki Ultimate Geography deck! :)

Also, the Anki name is relatively recognisable.

**Reply by jeprecated** [reply]
Agreed! And I have to admit my blunder that I somehow forgot that _Anki is currently in the name!_ :laughing: The current repo name as an org would be fine with me :+1:

**Comment by axelboc** [comment]
FYI, we can play with URL capitalisation- e.g. `UltimateGeography`

**Comment by axelboc** [comment]
This completely dropped off my mind... Re-reading our discussions, are we all on board with naming the org **anki-geo**? We could then name the main repo **ultimate-geography** and the future extension repos `ug-<name>` (for instance).

As discussed, _anki-geo_, means we can add other geography decks and sub-decks if we want to, and it doesn't prevent having "adapter repos" for other SRS systems, like `ultimate-geography-<srs-soft-name>`.

**Reply by jeprecated** [reply]
Sounds like a plan! :tada:

**Reply by jeprecated** [reply]
@aplaice and I both reply within 3 mins, lol :laughing:

**Reply by aplaice** [reply]
`anki-geo` is suitable as far as I'm concerned! It's short, topical, aspirational, but not crazily ambitious.

**Comment by axelboc** [comment]
Well, that was easy! 🎉 

- https://github.com/anki-geo
- https://github.com/anki-geo/ultimate-geography

**Reply by jeprecated** [reply]
The best thing about it is that https://github.com/axelboc/anki-ultimate-geography/ still forwards to the new repo :+1:

**Reply by axelboc** [reply]
Me before clicking on "Transfer": :cold_sweat: 😢 ... and after:  :relieved: 😎 :heart_eyes: 🍺 🎉 🙌

**Reply by axelboc** [reply]
> The best thing about it is that https://github.com/axelboc/anki-ultimate-geography/ still forwards to the new repo 👍

Yeah, we can take our time updating the links in the deck description and in other places, and announce the move in the next release notes. It transfers "watchers" and "stars" and everything... Really cool.
